stable/elasticsearch/README.md

```diff
@@ -121,6 +125,7 @@ The following table lists the configurable parameters of the elasticsearch chart
 | `data.terminationGracePeriodSeconds` | Data termination grace period (seconds)                             | `3600`                                              |
 | `data.antiAffinity`                  | Data anti-affinity policy                                           | `soft`                                              |
 | `data.nodeAffinity`                  | Data node affinity policy                                           | `{}`                                                |
+| `dataTypes`                          | Array with different data node types (see `data.` properties)       | `[]`                                                |
```

Comment:
Not clear to me, maybe state: `In case you want different data node types, optional array defining them. See documentation below.`
This PR introduce complex things and can be difficult to understand for not experienced people, we should make sure it stay simple.

Reply:
I've added a description more in line with the others. Please let me know if it's still not clear